### PR TITLE
Add ENTINA printer definitions

### PR DESCRIPTION
**Pull request by cgobat**
This PR adds printer defintions for the [ENTINA TINA2](https://www.entina3d.com/products/entina-mini-3d-printers-tina-2-basic) and [ENTINA X40](http://wiki.entina3d.com/doku.php/x40/index) 3D printers and their clones.

**Scope:**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Printer definition file(s)
- [ ] Translations

# How Has This Been Tested?

All definition files load and behave as expected in local Cura installation.

**Test Configuration**:
* Operating System: Windows 11
* This PR fixes the test failures of #14900, which was originally tested on an older Cura version

# Checklist:

- [x] My code follows the style guidelines of this project as described in [UltiMaker Meta](https://github.com/Ultimaker/Meta) and [Cura QML best practices](https://github.com/Ultimaker/Cura/wiki/QML-Best-Practices)
- [x] I have read the [Contribution guide](https://github.com/Ultimaker/Cura/blob/main/contributing.md) 
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have uploaded any files required to test this change

### CODE REVIEW

**Comment by cgobat** [comment]
Could I have some clarification about these test failures? It is throwing an `AssertionError: Unnecessary default_values for [...]`, for fields that I *do* want to actually define default values for. Why is this a problem/not allowed?

**Comment by jellespijker** [comment]
Please take a look at the failing unit tests: https://github.com/Ultimaker/Cura/actions/runs/4487096515/jobs/7890198991?pr=14926#step:13:446

**Comment by jellespijker** [comment]
because the definition from which you inherit, already sets a `default_value`,  easy solution is to change then into `value`.

**Comment by cgobat** [comment]
> easy solution is to change then into `value`.

You mean just change all of those `"default_value"` keywords to `"value"`?

**Comment by nallath** [comment]
Hey @cgobat Could you still change those default_values to value? Thanks!

**Comment by cgobat** [comment]
Related: #13110, #15095, #16045

**Comment by jellespijker** [comment]
@wawanbreton and/or @saumyaj3 can either of you take a look at this PR?

File: resources/definitions/entina_x40.def.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  "name": "ENTINA X40",
+  "inherits": "weedo_base",
+  "metadata":
+  {
+    "visible": true,
+    "author": "ENTINA",
+    "manufacturer": "ENTINA",
+    "file_formats": "text/x-gcode",
+    "machine_extruder_trains": {
+      "0": "weedo_extruder_left_x40",
```
Comment (saumyaj3) [inline resources/definitions/entina_x40.def.json:12]: These extruders are not available in the database.. Please add them too

File: resources/definitions/entina_x40.def.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      "1": "weedo_extruder_right_x40"
```
Comment (saumyaj3) [inline resources/definitions/entina_x40.def.json:13]: this one too

File: resources/definitions/weedo_f192.def.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    "inherits": "weedo_base",
+    "metadata": {
+        "visible": true,
+        "author": "WEEDO",
+        "manufacturer": "WEEDO",
+        "file_formats": "application/x3g",
+        "machine_extruder_trains":
+        {
+            "1": "weedo_extruder_left",
+            "0": "weedo_extruder_right"
```
Comment (saumyaj3) [inline resources/definitions/weedo_f192.def.json:13]: These extruders do not exist . please add them

**Review by saumyaj3 — CHANGES_REQUESTED** [review]
I see the extruder used in the printers are not available in the PR. Please create them and add it to this PR?

File: resources/definitions/entina_x40.def.json (hunk 1 above)
Comment (cgobat) [reply resources/definitions/entina_x40.def.json]: Thanks, good catch. Fixed now.

Should there be a unit test/check to catch things like undefined extruders and prevent them from slipping through?

**Pull request closed without merging**